lua/weapons/d4-so called Life Disruptor

```diff
@@ -0,0 +1,69 @@
+
+SWEP.PrintName			= "d4v3's so called Life Disruptor"
+SWEP.Author			    = "d4v3" 
+SWEP.Instructions		= "LMB TO SHOOT, RMB CREATES SHIELD"
+SWEP.ViewModel = "models/weapons/c_irifle.mdl"
+SWEP.WorldModel = "models/weapons/w_irifle.mdl"
+SWEP.ShootSound = Sound ( "Weapon_AR2.Single" )
+SWEP.Category = TASWeapons.Category
+SWEP.Primary.ClipSize = 25
+SWEP.Primary.DefaultClip = 60
+SWEP.Secondary.ClipSize = 5
+SWEP.Secondary.Ammo = ""
+SWEP.Secondary.DefaultClip = 5
+SWEP.Primary.Ammo = "AR2AltFire"
+SWEP.AdminSpawnable = true
```

Comment:
"AdminSpawnable" is not a SWEP property used by Gmod, and you don't use it anywhere else. Could this be a mistake?